src/eos-google-chrome

```diff
@@ -56,8 +56,8 @@ class GoogleChromeLauncher:
             logging.info("Flatpak launcher for Chrome found. Launching...")
             self._run_chrome_app(chrome_launcher, self._params)
         else:
-            logging.info("Could not find flatpak launcher for Chrome. Opening App Center...")
-            self._run_app_center_for_chrome()
+            logging.info("Could not find flatpak launcher for Chrome. Running installation script...")
+            self._install_chrome()
```

Comment:
I like more the second idea (because it will bring us the same result with less duplication).